From 4cccb82d5ba3612c95b7148c9f989ca3334f66ed Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 18 Dec 2021 15:17:34 +0000
Subject: [PATCH 089/115] Revert "Disable PixelProps when
 PLATFORM_SECURITY_PATCH_OVERRIDE is set (#714)"

This reverts commit 93aebed6207326f6faf39bced839aac344f094dd.
---
 core/java/android/app/Instrumentation.java | 14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

diff --git a/core/java/android/app/Instrumentation.java b/core/java/android/app/Instrumentation.java
index 52f15f7dbac..5f575033c2d 100644
--- a/core/java/android/app/Instrumentation.java
+++ b/core/java/android/app/Instrumentation.java
@@ -1159,11 +1159,8 @@ public class Instrumentation {
         Application app = getFactory(context.getPackageName())
                 .instantiateApplication(cl, className);
         app.attach(context);
-        String patchCrDroid = Build.VERSION.SECURITY_PATCH_CRDROID;
-        if ("".equals(patchCrDroid)) {
-            String packageName = app.getPackageName();
-            PixelPropsUtils.setProps(packageName);
-        }
+        String packageName = app.getPackageName();
+        PixelPropsUtils.setProps(packageName);
         return app;
     }
     
@@ -1181,11 +1178,8 @@ public class Instrumentation {
             ClassNotFoundException {
         Application app = (Application)clazz.newInstance();
         app.attach(context);
-        String patchCrDroid = Build.VERSION.SECURITY_PATCH_CRDROID;
-        if ("".equals(patchCrDroid)) {
-            String packageName = app.getPackageName();
-            PixelPropsUtils.setProps(packageName);
-        }
+        String packageName = app.getPackageName();
+        PixelPropsUtils.setProps(packageName);
         return app;
     }
 
-- 
2.25.1

